File: Funciones/8. mayor_dos_numeros/js/mayor.js
/**
 * Mayor de dos números  
 * Autor: Santiago GM
 * 03/04/2024
*/

let numUno;
let numDos;

//como paramétro
function mayor(pnumUno, pnumDos){
    numUno = pnumUno;
    numDos = pnumDos;

    if(numUno> numDos){
      return "El numero uno es mayor "+numUno
    }else{
        return "El numero dos es mayor " + numDos
    }
  
}

//como Expresión
const mayorExp= function (pnumUno, pnumDos){
  numUno = pnumUno;
  numDos = pnumDos;

  if(numUno> numDos){
    return "El numero uno es mayor "+numUno
  }else{
      return "El numero dos es mayor " + numDos
  }

}

File: python/10. Mayor_igual_numeros/utils/mayor.py
# Mayor igual de dos números
#Autor: Santiago GM

#Como paramétro

def validar_numeros(num_uno_p, num_dos_p) :
  num_uno, num_dos = num_uno_p, num_dos_p

  if (num_uno == num_dos) :
    return "Los número son iguales"
  elif (num_uno > num_dos) :
    return f"El número uno es el mayor {num_uno}"
  else :
    return f"El número dos es el mayor {num_dos}"

# Como Expresión

validar_numeros_lamb = lambda num_uno, num_dos : f"El número uno es el mayor {num_uno}" if num_uno > num_dos else f"El número dos es el mayor {num_dos}" if num_uno < num_dos else "Los números son iguales"